src/SFA.DAS.EmployerApprenticeshipsService.Application.UnitTests/Queries/GetEmployerAccountPaymentsByDate/WhenIGetEmployerAccountPaymentsByDate.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EAS.Application.Queries.GetEmployerAccountPaymentsByDate;
+using SFA.DAS.EAS.Application.Validation;
+using SFA.DAS.EAS.Domain.Data.Repositories;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+
+namespace SFA.DAS.EAS.Application.UnitTests.Queries.GetEmployerAccountPaymentsByDate
+{
+    public class WhenIGetEmployerAccountPaymentsByDate : QueryBaseTest<GetEmployerAccountPaymentsByDateHandler, GetEmployerAccountPaymentsByDateQuery, GetEmployerAccountPaymentsByDateResponse>
+    {
+        private Mock<ITransactionRepository> _transactionsRepository;
+
+        public override GetEmployerAccountPaymentsByDateQuery Query { get; set; }
+
+        public override GetEmployerAccountPaymentsByDateHandler RequestHandler { get; set; }
+
+        public override Mock<IValidator<GetEmployerAccountPaymentsByDateQuery>> RequestValidator { get; set; }
+
+        private const long ExpectedAccountId = 4564878;
+        private const string ExpectedUserId = "123dfv";
+
+        private readonly DateTime _fromDate = DateTime.Parse("01/01/2000");
```

Comment:
This is dependent on the locale of the user running the tests - if we're building a new date should we use ParseExact or better in this case use one of the DateTime constructors?

Reply:
I'll change this now, its another piece of code pulled over.